include/gc/Transforms/Passes.td

```diff
@@ -74,8 +74,11 @@ def IterativeTilingAndFusion : Pass<"iterative-tiling-and-fusion",
     Option<"useCostModel", "use-cost-model", "bool",
            /*default=*/"false",
            "Decide if enable cost model to control iterative fusion.">,
+    Option<"defaultNDTile", "default-nd-tile", "int",
+           /*default=*/"2",
+           "Set default ND-Tile size, such as 2[default] or 3.">,
```

Comment:
What's an ND tile?

Reply:
`NDTile` decides the maximum of increment variable `nonOneTileDims` as said above. This knob controls the granularity of tiling or fusion.

Reply:
I think this needs a better name and description. The comment in the knob says it's a tile size. I'd expect something like a [32, 16] as a value for an ND-tile size. A `2` for ND-tile size is not intuitive.

Reply:
`NDTile` is referring to `ndarray` in numpy. As for its description, in avoid of confusion, let me try to rephrase this description to "set default amount of non-one dimensions in `tileSize`"?